### ISSUE TITLE
FDSNWS does not download any data

### ISSUE THREAD

**Issue by ftilmann** [EIDA/webdc3#20]
In the download window (see screenshot) for FDSNWS I cannot figure out what to do next to actually download data. If I actually click on the individual buttons for the different EIDA nodes, every line seems to be an ERROR, e.g.
KO    LFK    --    HHN    2018-09-28T10:14:30.005380    2018-09-28T10:17:30.005380    ERROR
As a user I am completey stuck now. (this example is for a request for data from Turkish stations for the 28 September 2018 Palu earthquake (origin time 10:02.44, small window around P wave); as you see different networks, not sure why they should all result in an error. 
Error message is not helpful in identifying the cause. 
![Screenshot_2019-08-05 ORFEUS Data Center WebDC3 Web Interface-1](https://user-images.githubusercontent.com/7870220/62466809-83737180-b792-11e9-9439-917c168a3a8e.png)

**Comment by javiquinte** [comment]
The "Save" button is disabled all the time, isn't it? Is the error message displayed on the page? Or is it on the "Console" tab?

**Comment by ftilmann** [comment]
The error message is displayed on this page after clicking one of the grey boxes for the different data centres. I get a view as in the following screenshot - the "ERROR" is not visible in the screenshot but it would be there when scrolling the overlay window to the right.  (sorry for not including this in the first place)
![Screenshot_2019-08-05 ORFEUS Data Center WebDC3 Web Interface-2](https://user-images.githubusercontent.com/7870220/63031973-40617e80-beb5-11e9-9c9d-e11dcf9d6608.png)

**Comment by andres-h** [comment]
On 08/14/2019 04:47 PM, Javier Quinteros wrote:
> Assigned #20 <https://github.com/EIDA/webdc3/issues/20> to @andres-h 
This is ODC implementation, isn't it?

**Comment by javiquinte** [comment]
Yes, this is the deployment at ODC. That's why I assigned @jbienkowski 
I included you (@andres-h ) just because of your experience in the development when migrating to WS.

**Comment by andres-h** [comment]
On 08/14/2019 11:09 PM, Javier Quinteros wrote:
> Yes, this is the deployment at ODC. That's why I assigned @jbienkowski 
> <https://github.com/jbienkowski>
> I included you (@andres-h <https://github.com/andres-h> ) just because 
> of your experience in the development when migrating to WS.
Sorry, I understood that this was assigned only to me.
